### PR TITLE
Extra pronto usage examples in README [ci skip]

### PR DESCRIPTION
**Pull request by eliotsykes**
**Draft** extra usage examples. If there's a better spot in the README for these or they need work please let me know. I'd like to do what I can to help.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Changes
+
+* Extra pronto run usage examples in README
```
Comment (mmozuras) [inline CHANGELOG.md]: No need to update README.

**Comment by mmozuras** [comment]
Looks good :+1:

**Comment by eliotsykes** [comment]
Thanks @mmozuras - edit to changelog undone.

**Comment by mmozuras** [comment]
@eliotsykes no, thank you :bow: :smile:

**Pull request merged**